v4/source/posts.yaml

```diff
@@ -397,6 +397,33 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/channels/{channel_id}/posts/unread':
+    get:
+      tags:
+        - posts
+      summary: Get posts for a channel around last unread
+      description: |
```

Comment:
Can you add a note that this route will only be available in version 5.3 and later?